internal/ui/restore/json.go

```diff
@@ -99,6 +113,12 @@ type statusUpdate struct {
 	BytesSkipped   uint64  `json:"bytes_skipped,omitempty"`
 }
 
+type errorUpdate struct {
+	MessageType string `json:"message_type"` // "error"
+	Error       string `json:"error"`
+	Item        string `json:"item"`
+}
```

Comment:
Should this include a `During` field, like the backup errors do? Restore doesn't have different phases like the backup command does, but I wondered if consistency of JSON blobs across commands was valuable enough to include a `"during": "restore"` value (or similar).